test/functional/c/test_magnetic_symmetry.cpp

```diff
@@ -421,6 +424,60 @@ TEST(MagneticSymmetry, test_spgms_get_magnetic_dataset_high_mag_symprec) {
     if (dataset != NULL) spg_free_magnetic_dataset(dataset);
 }
 
+TEST(MagneticSymmetry, test_spg_get_magnetic_dataset_non_collinear) {
+    double lattice[3][3] = {{10, 0, 0}, {0, 10, 0}, {0, 0, 10}};
+    double position[][3] = {
+        {0, 0, 0},
+    };
+    int types[] = {1};
+    double spins[3] = {1, 0, 0};
+    int num_atom = 1;
+    SpglibMagneticDataset *dataset;
+    dataset = spg_get_magnetic_dataset(lattice, position, types, spins, 1,
+                                       num_atom, 1, 1e-5);
+
+    ASSERT_TRUE(dataset->n_operations == 16);
+    show_spg_magnetic_dataset(dataset);
+    spg_free_magnetic_dataset(dataset);
```

Comment:
```suggestion
    ASSERT_NEdataset, nullptr);
    EXPECT_EQ(dataset->n_operations, 16);
    if (testing::Test::HasFailure())
      show_spg_magnetic_dataset(dataset);
    spg_free_magnetic_dataset(dataset);
```
Please change `ASSERT` -> `EXCEPT`, otherwise `show_spg_magnetic_dataset` is meaningless.

Note: There is still an `ASSERT_NE` here because other further commands should fail or be meaningless otherwise.

Reply:
@lan496,I will include the change. OK?

Reply:
Sure. `show_spg_magnetic_dataset` was perhaps used in debugging.